src/routes/sessions/[uuid]/index.svelte

```diff
@@ -26,10 +26,9 @@
   }
 </script>
 
-<svelte:head>
-  <script crossorigin src="https://unpkg.com/webex@^2/umd/webex.min.js"></script>
-</svelte:head>
-
+{#if demo.isSDK || demo.isIC}
+  <SDKHeader />
```

Comment:
Please see previous comment on the component, otherwise it should be renamed to `SdkHeader`.